src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
-		https://index.docker.io \
-		-u <HEROKU_USERNAME> \
-		-p <HEROKU_PASSWORD> \
-		-N <DOCKER_REPO> \
-		-T <DOCKER_REPO_TAG>
+           Purpose of this email is to inform every team what
+           was release and to be abore in case of any breakage.
 
 
-Amazon
-------
+           # Release notes for v<VERSION>
 
-In order to make ``mozilla-releng/services`` work with Amazon following
-configuration needed to be in place:
+           <CONTENT-OF-RELEASE-PR>
 
-- 3 users are created (**RelEngDevelop**, **RelEngStaging**,
-  **RelEngProduction**) and their api credentials stored in taskcluster
-  secrets.
+           <LINK-TO-RELEASE-PR>
 
-- 3 custom policies were created (TODO) to allow above created users to sync 
 
-- 2 S3 buckets are created per each **src/*_frontend** application to hold
-  frontend code.
+           # Next release
 
-- For each S3 bucket a CloudFront Policy is created and configured to
+           Next release is going to be on
+           
+               YYYY-MM-DD MM:HH CET
+               YYYY-MM-DD MM:HH PST
 
-  - redirect every URL to ``index.html``
-  - redirect from http to https
+           and is going to be managed by
+           
+               <NEXT-RELEASE-MANAGER>
+           
+           You can follow the progress in release PR
 
-- A ``releng-cache`` S3 bucket is created and read/write/delete permission
-  granted for all above created policies. This S3 bucket is used to hold binary
-  files of builds.
+               <LINK-TO-NEXT-RELEASE-PR>
 
-- S3 buckets need to be created before deploying new service
 
-TODO: write how certificates are handled once Route51 is configured
 
+           [1] https://github.com/mozilla-releng/services
 
-Heroku
-------
 
-We currently use a (quite some time in beta) Heroku feature that allows to run
-docker images on Heroku infrastructure. To manually push docker images build in
-:ref:`previous praragraph <deploying-docker>` you need to push docker images to
-Heroku's custom docker registry.
+.. _`Example of release PR`: https://github.com/mozilla-releng/services/pull/237
+.. _`Open next release PR`: https://github.com/mozilla-releng/services/compare/production...master
+.. _`Release Enginnering`: https://wiki.mozilla.org/ReleaseEngineering
+.. _`Release Management`: https://wiki.mozilla.org/Release_Management
 
-.. code-block:: bash
 
-    % make build-tool-push
-	./result-tool-push/bin/push \
-		`realpath ./result-docker-releng_clobberer` \
-		https://registry.heroku.com \
-		-u <HEROKU_USERNAME> \
-		-p <HEROKU_PASSWORD> \
-		-N <HEROKU_APP>/web \
-		-T latest
+.. _release-managers:
+
+Release Managers
+----------------
+
+- `Rok Garbas`_
+
+
+.. _service-owners:
+
+Service Owners
+--------------
+
++---------------------------+---------------------------+
++ Service                   | Owner(s)                  |
++===========================+===========================+
++ :ref:`releng_archiver`    | - `Rok Garbas`_           |
```

Comment:
👍 That would be great. I'm happy we find people to be owners right now :) Maybe a compromise could be to contact release@mozilla.com?